spec/rspec/matchers_spec.rb

```diff
@@ -24,6 +24,16 @@
       end
     end
   end
+
+  describe "#respond_to?" do
+    it "returns true for methods in example scope" do
+      expect(self.respond_to?(:be_happy)).to eq(true)
```

Comment:
This can use the `respond_to` matcher, which is particiularly useful when checking multiple messages (as I suggest below that you do here):

``` ruby
expect(self).to respond_to(:be_happy, :have_eyes_closed)
```